src/scss/components/_button.scss

```diff
@@ -96,6 +110,30 @@
     width: 100%;
   }
 
+  &--loading {
+    position: relative;
+    transition: all .3s ease;
+    font-size: 0 !important;
```

Comment:
@eduardojmatos I think this is the case I discussed with you about overriding BEM classes.
BEM introduces a high level of specificity, if you need to override something